src/loaders/lottie/tvgLottieLoader.cpp

```diff
@@ -291,7 +291,7 @@ Paint* LottieLoader::paint()
 }
 
 
-bool LottieLoader::override(const char* slots, bool copy)
+bool LottieLoader::override(const char* slots, bool defaultSlot, bool copy)
```

Comment:
I don't think we need to change the API; it can handle this internally according to its policy.